wallet/concepts/apis.md

```diff
@@ -22,6 +22,13 @@ This API is specified by [EIP-1193](https://eips.ethereum.org/EIPS/eip-1193), an
 request users' Ethereum accounts, read data from blockchains the user is connected to, suggest
 that the user sign messages and transactions, and more.
 
+:::note
+MetaMask supports [EIP-6963](https://eips.ethereum.org/EIPS/eip-6963), which introduces an
+alternative discovery mechanism to the `window.ethereum` injected provider, enabling dapps to
+support [wallet interoperability](wallet-interoperabilty.md) and discover multiple injected wallet
+providers in a user's browser.
```

Comment:
enable wallet interoperability by ?
```suggestion
support [wallet interoperability](wallet-interoperabilty.md) by enabling the discovery of multiple injected wallet
providers in a user's browser.
```

idk, reads a little oddly to me. feel free to ignore